reference/shipping_provider.yml

```diff
@@ -1180,46 +1130,33 @@ components:
           format: int32
         attributes:
           type: array
-          description: A list of arbitrary properties stored as part of the product or
-            product variant meta fields. These consist of any public fields
-            specific to the carrier integration.
+          description: A list of arbitrary properties stored as part of the product or product variant meta fields. These consist of any public fields specific to the carrier integration.
           items:
             type: object
             properties:
               key:
                 type: string
-                description: The key associated with the product or product variant meta
-                  field
+                description: The key associated with the product or product variant meta field.
               value:
                 type: string
-                description: The value associated with the product or product variant meta
-                  field
+                description: The value associated with the product or product variant meta field
               namespace:
                 type: string
-                description: The namespace associated with
-                  product(/api-reference/catalog/catalog-api/product-metafields/createproductmetafield)
-                  or product variant
-                  (/api-reference/store-management/catalog/product-variants-metafields/createvariantmetafield)
-                  meta fields. The meta field namespace should be saved under
-                  this format `shipping_carrier_{yourCarrierId}` otherwise it
-                  will not be recognized as an attribute.
+                description: 'The namespace associated with product(/api-reference/catalog/catalog-api/product-metafields/createproductmetafield) or product variant (/api-reference/store-management/catalog/product-variants-metafields/createvariantmetafield) meta fields. The meta field namespace should be saved under this format `shipping_carrier_{yourCarrierId}` otherwise it will not be recognized as an attribute.'
```

Comment:
```suggestion
                description: 'The namespace associated with meta fields for [products](/api-reference/catalog/catalog-api/product-metafields/createproductmetafield) and [product variants](/api-reference/store-management/catalog/product-variants-metafields/createvariantmetafield). Save the meta field namespace using the format `shipping_carrier_{yourCarrierId}`; otherwise, it will not be recognized as a valid attribute.'
```